packages/core/src/client/index.ts

```diff
@@ -245,37 +273,43 @@ export class UploadAPI {
       getCancelURI,
       onChange,
       onError,
+      autoUpload,
```

Comment:
Can be slightly simplified below if you assign default `autoUpload = true` here.